qa/L0_inferentia_perf_analyzer/test.sh

```diff
@@ -26,7 +26,16 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # First need to set up enviroment
-source ${TRITON_PATH}/python_backend/inferentia/scripts/setup.sh -p
+if [ ${USE_TENSORFLOW} == "1" ] ; then
```

Comment:
Having two different options will be better USE_TENSORFLOW and USE_PYTORCH. You should check exactly one of them is non-zero.. 1 in this case.. The TF version should be another option in future..